modules/nw-egress-router-redirect-mode-ovn.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc
+
+[id="nw-egress-router-redirect-mode-ovn_{context}"]
+= Deploying an egress router pod in redirect mode
+
+You can deploy an egress router pod to redirect traffic from its own reserved source IP address to one or more destination IP addresses.
+
+After you add an egress router pod, the client pods that need to use the reserved source IP address must be modified to connect to the egress router rather than connecting directly to the destination IP.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a network attachment definition.
+
+. Create an egress router pod.
+
+. To ensure that other pods can find the IP address of the egress router pod, create a service that uses the egress router pod, as in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: egress-1
+spec:
+  ports:
+  - name: http
+    port: 80
+  - name: https
+    port: 443
+  type: ClusterIP
+  selector:
+    name: egress-router-pod
+----
++
+After you create the service, your pods can connect to the service. Their connections are redirected to the corresponding ports on the destination IP address and the connections originate from the reserved source IP address.
+
+.Verification steps
+
+To verify that the egress router pod started and Multus attached a second network interface, complete the following procedure:
```

Comment:
maybe;

s/second/secondary/?

Reply:
If secondary is consistent with elsewhere, then certainly.  One thought this triggers is that I don't think I indicated that a new default route is added and the route uses the primary (?) interface, `eth0`.

Reply:
Well, the route thing is a different topic, please see: https://github.com/openshift/egress-router-cni/pull/27
but you can check that the routes will be as follows

```
default via 192.168.111.1 dev net1 
10.128.2.0/23 dev eth0 proto kernel scope link src 10.128.2.164 
192.168.111.0/24 dev net1 proto kernel scope link src 192.168.111.200 
192.168.111.1 dev net1
```

being `192.168.11.1` the gateway and `net1`the macvlan interface

Reply:
As a noob, the output of `ip ro show` was clearest to me.  The only way that I know to show that output is `oc exec -it...` but A) doesn't that add some tainted cooties to the pod? and B) I'm unsure if that teeters from verification into troubleshooting.  I'll noodle with Jason or other writers if I find that my understanding of "A" is incorrect.

Reply:
What you could do this time to get to the `ip ro show`output is just `oc rsh` the actual pod ip, and then run the command. You can also go to the network namespace, enter it and run the command from the host as well. If you feel that that would help verification I'm happy to provide detailed commands (which IIRC are in the developer's guide as well). TL; DR the idea about the routing is, everything but the cluster network, out using the provided gateway, and that gateway out  using the macvlan interface.